ps_linklist.php

```diff
@@ -136,34 +132,32 @@ private function installFixtures()
     {
         $installed = true;
         $errors = $this->getRepository()->createTables();
-        if (!empty($errors)) {
-            $this->addModuleErrors($errors);
+        if (false === $errors || (is_array($errors) && !empty($errors))) {
```

Comment:
If there is no errors (`false === $errors`), `$installed = false;` :thinking: 

redundant check for `is_array`

Reply:
The problem is that there are two possible repositories (legacy one and new one), one returns a boolean and the other an array. So there was a problem with `!empty($errors)` that was true when the createTable function returned `true` (on success)

Which is why I need to check if `$errors` is an array, as for `addModuleErrors` it only accepts array or else it fails, hence the second `is_array` check The code can probably be optimized but both checks are useful/needed